tests/components/homekit_controller/snapshots/test_init.ambr

```diff
@@ -26,6 +26,7 @@
         'model': 'AP2',
         'name': 'Airversa AP2 1808',
         'name_by_user': None,
+        'primary_config_entry': 'TestData',
```

Comment:
Why has `homekit_controller` as the only integration a primary config entry set?

Reply:
I don't understand the question. The PR has tons of changed snapshots due to the addition of `primary_config_entry`, there's nothing special about homekit_controller?

Reply:
`MockConfigEntry` allows setting the entry id. Some integrations to that